src/wallet/components/wallet-advanced-send/components/Send.tsx

```diff
@@ -0,0 +1,72 @@
+import { Skeleton } from '@/internal/components/Skeleton';
+import { background, border, cn, color } from '@/styles/theme';
+import { ETH_REQUIRED_FOR_SEND } from '../constants';
+import type { SendReact } from '../types';
```

Comment:
thought: Unrelated to this PR, just thinking out loud, but it would make more sense to me for component props to live alongside the component, and be exported from the main types file.

Reply:
@dgca sorry, I'm a bit confused.

> but it would make more sense to me for component props to live alongside the component

> and be exported from the main types file.

these seem contradictory. did you mean the former or the latter?

our (not super-strict) convention has been to create a types file that has the props type for the top-level component, as well as any reused types. we've discussed changing that, definitely open to your feedback

Reply:
My bad, yeah I meant the former

e.g.

```tsx
// MyComponent.tsx
export type MyComponentProps = { name: string };

export function MyComponent({ name }: MyComponentProps) {
  return <p>Hello {name}</p>
}
```

Then from the index file, we can

```tsx
export { MyComponent, MyComponentProps } from './components/MyComponent';`
```